source/_integrations/tag.mqtt.markdown

```diff
@@ -16,6 +16,11 @@ MQTT scanners are only supported through [MQTT discovery](/docs/mqtt/discovery/)
 The discovery topic needs to be: `<discovery_prefix>/tag/[<node_id>/]<object_id>/config`.
 
 {% configuration %}
+encoding:
```

Comment:
Not implemented for `tag`, only supported through discovery which is utf-8 only.